src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -169,13 +168,20 @@ export const MultiSelect = chakra(
         return item.isDisabled;
       };
 
+      // Logic for isSearchable MultiSelect
```

Comment:
This is no longer correct so you can remove the comment. It's no longer logic and just the component.

Reply:
Adjusted